src/CMake/VisItFunctions.cmake

```diff
@@ -120,3 +127,89 @@ function(SETUP_APP_VERSION name ver_ascii )
 endfunction()
 
 
+function(visit_install_system_libs)
+    #-------------------------------------------------------------------------
+    # Install the system libraries required by Qt to the lib directory.
+    # The versions of these libraries change frequently, so including
+    # them means VisIt may continue to run even if the OS is upgraded.
+    # "/lib64" is for rhel7 and fedora31, "/usr/lib/x86_64-linux-gnu"
+    # is for ubuntu 18, debian 10 and debian 11, "/lib/x86_64-linux-gnu"
+    # is for ubuntu 20.
+    #-------------------------------------------------------------------------
+    if(EXISTS "/lib64/libicui18n.so")
```

Comment:
Just curious but does this mean were making copies of the system libs at VisIt's *install* point so we have em as a fallback when/if the system gets updated?

Cool! 💪🏻

Reply:
Yes, @brugger1 did the work, I just moved the logic from root CMakeLists.txt and made it a function.